WordPress/wporg-theme-directory
# Remove the Favorite API & block, use shared block

This removes the favorite block & API in favor of the new shared favorite button. The API endpoints are not necessary with the shared block's endpoint + callback process, so I've removed them here.

See https://github.com/WordPress/wporg-mu-plugins/discussions/617

**To test**

- Use the branch here https://github.com/WordPress/wporg-mu-plugins/pull/635
- Make sure to build both projects
- View the theme directory
- There are no favorite buttons on the grid view
- View a single theme
- If you're logged in, you should see a button in the theme header
- You should be able to click or trigger with a keyboard to favorite/unfavorite items, and the button should update with the interaction
- Reload the page or view /browse/favorites to ensure the favorite saved correctly
- If you are not logged in, there is no favorite button
